src/pages/contributing/component/index.mdx

```diff
@@ -521,19 +521,219 @@ leaving any other comments on
+
+</Column>
+</Row>
+
+## Interactive state color
+
+<!--
+Use a table to document properties and color tokens for interactive states. Begin the table with the State column, not the Element column; this helps users scan faster and better reference accompanying images, which are also organized by state. You can reference this [table generator](https://www.tablesgenerator.com/markdown_tables) for easy table creation.
+
+You may also choose to include images or tabs to illustrate variants.
+
+The following table shows an example from [Accordion](https://carbondesignsystem.com/components/accordion/style).
+-->
+
+| State    | Element | Property   | Color token      |
+| -------- | ------- | ---------- | ---------------- |
+| Hover    | Header  | background | `$layer-hover`   |
+| Focus    | Header  | border     | `$focus`         |
+| Disabled | Header  | border-top | `$border-subtle` |
+| Disabled | Title   | background | `$text-disabled` |
+| Disabled | Icon    | fill       | `$icon-disabled` |
+
+<Row>
+<Column colLg={12}>
+
+<Tabs>
+
+<Tab label="Default">
+
+![default accordion alignment interactive states](images/accordion-style-2.png)
+
+</Tab>
+
+<Tab label="Flush">
+
+![flush accordion alignment interactive states](images/accordion-style-3.png)
+
+</Tab>
+
+</Tabs>
+
+</Column>
+</Row>
+
+## Typography
+
+<!--
+Begin this section with any brief content guidelines. Link out to additional guidelines if needed.
+
+Then, use a table to document font-size, font-weight, and type tokens in this section. You can reference this [table generator](https://www.tablesgenerator.com/markdown_tables) for easy table creation.
+
+The following table shows an example from [Accordion](https://carbondesignsystem.com/components/accordion/style).
+-->
+
+All accordion titles are set in sentence case. See the accordion
+[content guidelines](/components/accordion/usage#content) for more details.
+
+| Element | Font-size (px/rem) | Font-weight   | Type token |
+| ------- | ------------------ | ------------- | ---------- |
+| Title   | 14 / 0.875         | Regular / 400 | `$body-01` |
+| Content | 14 / 0.875         | Regular / 400 | `$body-01` |
+
+## Structure
+
+<!--
+Begin the structure section with a table to document states, spacing, and any other component-specific configurations. You can reference this [table generator](https://www.tablesgenerator.com/markdown_tables) for easy table creation.
+
+Add supporting images to this section, with at least one image showing structure and another showing spacing measurements. Images should be accompanied by a descriptive caption and relate to the spec charts. Note we use the `image--fixed` markdown component in the Structure section to maintain size accuracy.
+
+Variants: You may need to provide additional specs for component variants like default versus flush alignment. For each variant, make a new h3 section and give it a name. The following are example section headers:
+Default style text area structure
```

Comment:
I think we might be able to take out the text area H3 examples here if the idea is just showing the naming convention of how to construct `style` H3s versus `alignment` H3s, because it's a bit redundant with text input.

If that is not the case and it's more about showing possible H3s for this section that could appear on one given style tab, then maybe separate the structured list H3s from the text input/area H3s with a space to break them up more? Let me know if that makes sense.

Reply:
Good point. My intention was about showing possible H3 for the section, including style and alignment. I'll add some returns to show the pairings more clearly.